src/config.rs

```diff
@@ -204,6 +210,32 @@ impl Config {
         self.search = Some(search);
         self.last_search = LastSearch::Search;
     }
+
+    /// Normalize config.
```

Comment:
The docstring should document current behavior more specifically.

Also I'm thinking by emitting warnings somehow, still don't want to depend on logging. So let's keep bare normalization for now, and will add another method with warnings in future.